main.tf

```diff
@@ -128,7 +128,7 @@ resource "aws_instance" "default" {
 
 module "dns" {
   source  = "cloudposse/route53-cluster-hostname/aws"
-  version = "0.7.0"
+  version = "0.9.0"
```

Comment:
@sfc-gh-pkommini `route53-cluster-hostname` supports context, so please, add
`context = module.this.context` at the end of arguments for module call

Reply:
ACK.

Reply:
@maximmi Seems like these new changes are causing some errors. Maybe I should revert them?

Reply:
no, those are ok, the errors are unrelated

Reply:
I see. Will push the changes again.